### PR TITLE
Extend installation guide with two more options

### PR DESCRIPTION
**Pull request by lukas-bednar**
* adding section about installation from APB
* adding section about kubevirt-ansible

### CODE REVIEW

File: installation.md [hunk 1]
```diff
@@ -96,6 +101,22 @@ Once privileges are granted, the `kubevirt.yaml` can be deployed:
...
+In order to do that please follow documentation of
+[KubeVirt APB repository](https://github.com/ansibleplaybookbundle/kubevirt-apb).
+
+### Using Ansible playbooks
+
+There is project [kubevirt-ansible](https://github.com/kubevirt/kubevirt-ansible)
+which provides a collection of playbooks to install KubeVirt on OpenShift or
+Kubernetes cluster.
+
+Supported scenarions
```
Comment (lukas-bednar) [inline installation.md]: I am considering to remove this section, because scenarios are described in kubevirt-ansible repository.

**Comment by lukas-bednar** [comment]
Hi @fabiand could you please take a look at this simple change, I added two more references to other installation options.

File: installation.md [hunk 2]
```diff
@@ -96,6 +101,18 @@ Once privileges are granted, the `kubevirt.yaml` can be deployed:
...
+
+You can find KubeVirt in OpenShift Service Catalog and install it from there.
+In order to do that please follow documentation of
+[KubeVirt APB repository](https://github.com/ansibleplaybookbundle/kubevirt-apb).
+
+### Using Ansible playbooks
+
+There is project [kubevirt-ansible](https://github.com/kubevirt/kubevirt-ansible)
+which provides a collection of playbooks to install KubeVirt and it's related
+components on top of OpenShift or Kubernetes cluster.
```
Comment (rmohr) [inline installation.md]: s/cluster/clusters/ or just remove `cluster`.

File: installation.md [hunk 3]
```diff
@@ -77,6 +77,11 @@ The most recent version of the tool can be retrieved from the [official release
 page](https://github.com/kubevirt/kubevirt/releases).
 
 ## Deploying on OpenShift
+
+There are three following ways how to deploy KubeVirt on OpenShift
```
Comment (rmohr) [inline installation.md]: There exist three ways on how to deploy KubeVirt on OpenShift.

File: installation.md (hunk 2 above)
Comment (lukas-bednar) [reply installation.md]: Addressed

File: installation.md (hunk 3 above)
Comment (lukas-bednar) [reply installation.md]: Addressed

**Comment by lukas-bednar** [comment]
@rmohr I addressed your comments, thanks!

File: installation.md [hunk 4]
```diff
@@ -96,6 +101,18 @@ Once privileges are granted, the `kubevirt.yaml` can be deployed:
 kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt.yaml
 ```
 
+### From Service Catalog as an APB
+
+You can find KubeVirt in OpenShift Service Catalog and install it from there.
+In order to do that please follow documentation of
```
Comment (rmohr) [inline installation.md]: follow `the` documentation

Comment (lukas-bednar) [reply installation.md]: Addressed

**Pull request merged**